packages/react-native-builder-bob/src/targets/codegen.ts

```diff
@@ -0,0 +1,42 @@
+import path from 'path';
+import fs from 'fs-extra';
+import spawn from 'cross-spawn';
+import type { Input } from '../types';
+import { patchCodegen } from '../utils/patchCodegen';
+
+type Options = Input;
+
+export default async function build({ root, report }: Options) {
+  try {
+    const packageJsonPath = path.resolve(root, 'package.json');
+    if (!(await fs.pathExists(packageJsonPath))) {
+      throw new Error(
+        `Couldn't find a 'package.json' file in '${root}'. Are you in a project folder?`
+      );
+    }
+
+    spawn.sync('npx', ['react-native', 'codegen'], {
+      stdio: 'inherit',
+    });
+
+    patchCodegen(root);
+
+    report.success('Codegen patched successfully!');
```

Comment:
Let's change the message since the message after the whole generation process and not only the patching

```suggestion
    report.success('Generated native code with codegen');
```